### PR TITLE
AdoptAWidget: PageView

### LINKED ISSUE #69487: Adopt a Widget - PageView
The [PageView](https://master-api.flutter.dev/flutter/widgets/PageView-class.html) widget needs:

- A code / DartPad sample

For details about how to participate, see [flutter.dev/adoptawidget](https://flutter.dev/adoptawidget)

### PR DESCRIPTION
**Pull request by isinghmitesh**
# AdoptAWidget: PageView

## Description

Adds a dartpad sample to the PageView widget API Docs.

This pull request is:

- [x] A code sample
- [ ] More references
- [ ] More explanation

## Related Issues
Closes #69487

## Checklist

Before you create this PR, confirm that it meets all requirements listed below by checking the relevant checkboxes (`[x]`). This will ensure a smooth and quick review process.

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I signed the [CLA].
- [x] I read and followed the [Flutter Style Guide], including [Features we expect every widget to implement].
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] All existing and new tests are passing.
- [x] The analyzer (`flutter analyze --flutter-repo`) does not report any problems on my PR.
- [x] I am willing to follow-up on review comments in a timely manner.

### CODE REVIEW

File: packages/flutter/lib/src/widgets/page_view.dart [hunk 1]
```diff
@@ -572,6 +572,35 @@ const PageScrollPhysics _kPagePhysics = PageScrollPhysics();
 ///
 /// {@youtube 560 315 https://www.youtube.com/watch?v=J1gE9xvph-A}
 ///
+/// {@tool dartpad --template=stateless_widget_scaffold}
+///
+/// Here is an example of [PageView]. It creates three centered [Text] widgets as children for PageView.
+/// Which is scrollable horizontally.
```
Comment (Piinks) [inline packages/flutter/lib/src/widgets/page_view.dart:578]:
```suggestion
/// Here is an example of [PageView]. It creates a centered [Text] in each of the three pages
/// which scroll horizontally.
```

File: packages/flutter/lib/src/widgets/page_view.dart [hunk 2]
```diff
@@ -572,6 +572,35 @@ const PageScrollPhysics _kPagePhysics = PageScrollPhysics();
...
+///
+/// ```dart
+///  Widget build(BuildContext context) {
+///    final controller = PageController(initialPage: 0);
+///    return PageView(
+///      // scrollDirection property defaults to Axis.horizontal if not given explicitly
```
Comment (Piinks) [inline packages/flutter/lib/src/widgets/page_view.dart:584]:
```suggestion
///      /// [PageView.scrollDirection] defaults to [Axis.horizontal].
```

File: packages/flutter/lib/src/widgets/page_view.dart [hunk 3]
```diff
@@ -572,6 +572,35 @@ const PageScrollPhysics _kPagePhysics = PageScrollPhysics();
...
+///      // use Axis.vertical for scrolling vertically
```
Comment (Piinks) [inline packages/flutter/lib/src/widgets/page_view.dart:585]:
```suggestion
///      /// Use [Axis.vertical] to scroll vertically.
```

**Review by Piinks — COMMENTED** [review]
Hi @singhmitesh welcome! :tada:
Thank you for the contribution!

**Comment by singhmitesh** [comment]
> Hi @singhmitesh welcome! 
> Thank you for the contribution!

Very happy about my first contribution :smiley:

**Review by Piinks — APPROVED** [review]
Flutter_LGTM

**Review by johnpryan — APPROVED** [review]

**Comment by Piinks** [comment]
@singhmitesh Can you update your PR with the latest changes from master? The test failure here is unrelated to your change. :)

**Comment by singhmitesh** [comment]
> @singhmitesh Can you update your PR with the latest changes from master? The test failure here is unrelated to your change. :)

on it :+1:

**Comment by singhmitesh** [comment]
Hi @Piinks ,

As I see right now all tests have passed. What comes next? :)

**Comment by Piinks** [comment]
> Hi @Piinks ,
> 
> As I see right now all tests have passed. What comes next? :)

Now we merge! 🎉 😄

**Pull request merged**